client-cli/src/main/java/org/eclipse/dataspaceconnector/registration/cli/RegistrationServiceCli.java

```diff
@@ -28,6 +28,9 @@ public class RegistrationServiceCli {
     @CommandLine.Option(names = "-s", required = true, description = "Registration service URL", defaultValue = "http://localhost:8181/api")
     String service;
 
+    @CommandLine.Option(names = "-d", required = true, description = "Client DID")
```

Comment:
does this mean that we need to provide client DID to even get list of participants ?

Reply:
Well, this point has not been made very clear, but the issue states "When a participant interacts with Registration service". It seems reasonable that you have no business listing participants unless you're a member of a dataspace. @zeier ?

Reply:
@algattik there is no requirement for this, we have to choose how we want our RegistrationService to behave. It's reasonable to only allow dataspace members to list participants.